hr_commission/models/res_partner.py

```diff
@@ -11,6 +11,7 @@ class ResPartner(models.Model):
         selection_add=[("salesman", "Salesman (employee)")],
     )
     employee_id = fields.Many2one(
+        string="Employee.",
```

Comment:
Remove the final dot

Reply:
We got this problem so we need to add dot to make a different
```
2019-07-01 06:18:34,600 13216 WARNING odoo12_hr_commission odoo.addons.base.models.ir_model: Two fields (employee_id, employee) of res.partner() have the same label: Employee. 
2019-07-01 06:18:34,636 13216 WARNING odoo12_hr_commission odoo.addons.base.models.ir_model: Two fields (employee_id, employee) of res.users() have the same label: Employee. 
```

Reply:
And where does `employee` come from?

Reply:
employee from res_partner.py in module base.

Reply:
OK, that's a new field. We can call this one then `Employee reference`, but I would beg you to check a bit deeper now if there's another link in standard that makes the same function that this `employee_id` field.

Reply:
I checked and found nothing.

Reply:
OK, thanks, then the other label is better IMO

Reply:
@pedrobaeza thank for your reviews.